### PR TITLE
Improve example project and its documentation

### PR DESCRIPTION
**Pull request by janpio**
- update` example/README` and `.github/CONTRIBUTING.md`
- copy over `native-bridge.js` from `/core` to `/www` in `copy` script

also:
- rename example project from `avocado` to `capacitor-example`
- update package-lock.json files
- remove `@ionic/cli-plugin-cordova`
- update `@ionic/app-scripts` to a version that likes recent NPM on Windows

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
 
+## 3. Build the native Capacitor Apps
 
-## Build iOS App
+Now that everything is in place you can build the native Capacitor Apps:
 
-```
+### a) Build iOS App
+
+```bash
```
Comment (jcesarmobile) [inline example/README.md]: This is not bash, nor commands, so not sure if should be formated like this.
Also, should open IonicRunner.xcworkspace, not the .xcodeproj

Comment (janpio) [reply example/README.md]: - correct @ bash
- `xcodeproj` is what was already there. will fix.

**Comment by janpio** [comment]
Note:

I also hat to do the following to get the iOS project building:

```
cd ios/IonicRunner

pod repo update
pod update CapacitorCordova
pod update Capacitor
pod install
```

I am not 100% sure this was _all_ needed or just a local fluke, so I am not adding it to the document yet but just noting it down here.

**Comment by jcesarmobile** [comment]
pod install is needed, the others shouldn't, but if you needed them, maybe should be added.

**Comment by janpio** [comment]
Will do a new checkout tomorrow on my super slow Mac and check again. I think I got error messages on `pod install` because the lockfile was out of date I guess.

**Comment by janpio** [comment]
As planned I tested with a new checkout. Running `pod update` in `ios/IonicRunner` was actually enough to update the `Podfile.lock` and install the Pods into `/Pods` - so I added those as the additional iOS instructions.

After that I could successfully build and run the iOS project, so I think this should be good now.

I also improved the introduction paragraph a bit and explained more what this project is and how it works.

**Comment by mlynch** [comment]
Is this ready for merge review?

**Comment by janpio** [comment]
Yes, this contains all I know about the `example` project now @mlynch ;)

**Comment by mlynch** [comment]
🚀

**Pull request merged**